bin/utils.js

```diff
@@ -8,6 +8,7 @@
   space-before-function-paren
 */
 const open = require('opn');
+const portfinder = require('portfinder');
```

Comment:
Why not `get-port` https://github.com/sindresorhus/get-port

Reply:
This was already in use before I made any changes though...

Reply:
Hmm... get-port will simply fall back to random port instead of trying to look for the next available port in sequence. Although I am not sure if seeking ports in sequence was ever a requirement. This was the behavior before I added the retries, so I try to maintain that behavior.

Reply:
well, if you're sure, I can switch to get-port. Just need to confirm because portfinder was introduced by whoever introduced port finding in the first place. And normally I try not to change project dependencies for projects that I don't own.

Reply:
@u9520107 hm, If we used it before, I propose to leave it and maybe change it in the next version. Good work.

Reply:
Thanks!